src/controllers/stop.screen.controller.ts

```diff
@@ -75,19 +81,30 @@ const setContent = async (req: Request, stopType: string) => {
     const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
     var companyName = companyProfile.companyName;
     if (companyName === "") {
-        if(stopType === STOP_TYPE.DISSOLVED){
+        if(stopType === STOP_TYPE.DISSOLVED && lang === "en"){
```

Comment:
Would it be safer to check for "cy" even if ity's the same as other stop types?

Reply:
or 
if (stopType ===  STOP_TYPE.DISSOLVED){ 
 if (lang === "en") {
 } else {
 }
}
?

Reply:
sorry but I don't follow, is there a use case that is likely to be broken with the current logic?

Reply:
It's not very clear to me - is the cy case for DISSOLVED just the same as the other STOP_TYPES? That's what the code appear to do but not very clear.

Reply:
>  is the cy case for DISSOLVED just the same as the other STOP_TYPES?
Yes that is correct, in Welsh the words "This company" is always in the middle of the statement hence it will always use key `stopPageThisCompanyAtMiddleOfstatement` with the value "y cwmni hwn".